Rocket.Chat/Models/Message.swift

```diff
@@ -31,6 +31,14 @@ enum MessageType: String {
     case subscriptionRoleRemoved = "subscription-role-removed"
     case roomArchived = "room-archived"
     case roomUnarchived = "room-unarchived"
+
+    var sequential: Bool {
+        if self == .image {
+            return false
+        }
```

Comment:
I believe the type `video` could also be used to return `false`. What do you think?

Reply:
I can add that, but videos don't even show up here. Are they implemented?

Reply:
They show up, you need to upload a video file.

Reply:
#[Prevent video being sequential message](https://github.com/RocketChat/Rocket.Chat.iOS/pull/618/commits/9618ce82d685d4bfa45af35073d6d7613228654c)